static/app/views/starfish/modules/databaseModule/index.tsx

```diff
@@ -51,7 +50,8 @@ function DatabaseModule() {
   } = useQueryMainTable({
     transaction,
     table,
-    action,
+    filterNew,
+    filterOld,
```

Comment:
Ohh we just weren't passing this in, I may have broken this earlier after refactoring some stuff. Nice find!